### DISCUSSION TITLE
除了markdown之外，是否可以支持mdx格式

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7097] [Ideas]
不清楚halo能否支持或有相关的计划

**Comment by IshuaWang** [comment]
我重新看了一下，vue好像也可以支持mdx

**Comment by ruibaby** [comment]
MDX 无法在浏览器环境使用，需要在 Node 环境编译，估计不好实现。

**Reply by IshuaWang** [reply]
> MDX 无法在浏览器环境使用，需要在 Node 环境编译，估计不好实现。

我对这方面不太懂，可以把mdx文件直接在服务器端编译好吗？就像静态站点托管docusaurus一样。
